### PR TITLE
fix: update the github icon

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
## before
image

## after
image

### CODE REVIEW

File: src/ansys_sphinx_theme/__init__.py [hunk 1]
```diff
@@ -167,12 +167,26 @@ def setup_default_html_theme_options(app):
...
+        theme_options.setdefault(
             "navbar_end", ["version-switcher", "theme-switcher", "navbar-icon-links"]
         )
-    app.config.html_theme_options.setdefault("collapse_navigation", True)
-    app.config.html_theme_options.setdefault("navigation_with_keys", True)
+    theme_options.setdefault("collapse_navigation", True)
+    theme_options.setdefault("navigation_with_keys", True)
+
+    # update github icon links
+    # if icon links are not given override them with default links
```
Comment (jorgepiloto) [inline src/ansys_sphinx_theme/__init__.py:180]:
```suggestion
    # Update the icon links. If not given, add a default GitHub icon.
```

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css [hunk 2]
```diff
@@ -600,6 +600,10 @@ Navigation column (according to side column)
   display: unset;
 }
 
+.theme-switch-button span {
+  color: #fff;
```
Comment (germa89) [inline src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css:604]: Wouldn't it be better to use the ``--pst-color-primary`` variable? So you "forget about it" in the next theme changes?

**Review by germa89 — APPROVED** [review]
LGTM.

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css (hunk 2 above)
Comment (Revathyvenugopal162) [reply src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css:604]: I am using these variables for all items other than the navigation bar, which remains black (until we change its color). The primary color is different for both theme options, but we can define it to match the navigation color.

Comment (jorgepiloto) [reply src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/css/ansys_sphinx_theme.css:604]: Even with that, we should have a variable for basic colors. These could be later reused by the `--pst-color-primary` when declared for the light and dark themes.

For the moment, we can leave it as this. We are going to reformat the CSS very soon.

**Review by jorgepiloto — APPROVED** [review]

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Comment by Revathyvenugopal162** [comment]
Thank you all for the review.

**Pull request merged**